modules/ROOT/pages/users.adoc

```diff
@@ -75,6 +75,7 @@ include::partial$include-nav-steps-org-admin.adoc[]
 
 include::partial$include-nav-users.adoc[]
 
+. Click the name of the user to whom you want to assign permissions.
 . In the *Permissions* section, click *Add permissions*.
```

Comment:
You have to select the section, such as Anypoint Code Builder. And **Add permissions** is not there for me. You select them from Permission(s).
![image](https://github.com/mulesoft/docs-access-management/assets/19573371/244c83c1-999c-4cbc-a171-523730ccec08)

Reply:
Oh, these instructions are for after you switch to the new features. I missed that.